homeassistant/components/sensor/ted5000.py

```diff
@@ -22,13 +22,14 @@
 
 DEFAULT_NAME = 'ted'
 
-MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=10)
+MIN_TIME_BETWEEN_UPDATES = timedelta(seconds=3)
 
 
 PLATFORM_SCHEMA = PLATFORM_SCHEMA.extend({
     vol.Required(CONF_HOST): cv.string,
     vol.Optional(CONF_PORT, default=80): cv.port,
     vol.Optional(CONF_NAME, default=DEFAULT_NAME): cv.string,
+    vol.Optional('show_unavailable', default=True): cv.boolean,
```

Comment:
Please use a constant for `'show_unavailable'`.

Reply:
Which one? I looked everywhere and can't find one that suits this. Do you mean creating a local constant???

Reply:
Would CONF_DISPLAY_OPTIONS = 'display_options' be good?

Reply:
Removed the option as requested by someone else...